acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuBeaconNode.java

```diff
@@ -366,18 +344,32 @@ public void waitForBlsToExecutionChangeEventForValidator(final int validatorInde
 
     waitFor(
         () -> {
-          final List<tech.pegasys.teku.api.schema.capella.SignedBlsToExecutionChange>
-              blsToExecutionChanges =
-                  getEventsOfTypeFromEventStream(
-                      EventType.bls_to_execution_change, this::mapBlsToExecutionChangeFromEvent);
-          assertThat(blsToExecutionChanges.stream())
-              .anyMatch(m -> UInt64.valueOf(validatorIndex).equals(m.message.validatorIndex));
+          final List<Integer> blsToExecutionChanges =
+              getEventsOfTypeFromEventStream(
+                  EventType.bls_to_execution_change, this::getValidatorIdsFromBlsChange);
+          assertThat(blsToExecutionChanges.stream()).anyMatch(m -> validatorIndex == m);
         });
   }
 
-  private SyncingResponse fetchSyncStatus() throws IOException {
-    String syncingData = httpClient.get(getRestApiUrl(), "/eth/v1/node/syncing");
-    return JSON_PROVIDER.jsonToObject(syncingData, SyncingResponse.class);
+  private JsonNode fetchSyncingStatus() throws IOException {
+    final String syncingData = httpClient.get(getRestApiUrl(), "/eth/v1/node/syncing");
+    return OBJECT_MAPPER.readTree(syncingData).get("data");
+  }
+
+  private boolean fetchSyncStatusElOffline() throws IOException {
```

Comment:
nit: I'd either go for sync or syncing regarding the name of these methods as long as we keep them all the same